converter/diag/report.go

```diff
@@ -6,14 +6,47 @@ import (
 
 const Text = ".txt"
 
-// generateTextReport generates a text report for the diagnostics.
-func generateTextReport(writer io.Writer, ds Diagnostics) error {
-	content := ds.Error()
+const criticalErrorFooter = `
```

Comment:
Wordsmithing is welcome on these 3 if we like the concept of a footer under the diags. @clayton-cornell @rfratto 

Check out the new tests below for how this will print out in practice.